ui/src/app/views/dataset/component.tsx

```diff
@@ -31,17 +40,42 @@ export const styles = {
   }
 };
 
+const ResourceListItem: React.FunctionComponent <
```

Comment:
Nice helper component here! Do you think we will need these list items elsewhere on the site? It may be worth making it a common component instead of inside the dataset builder file.

Reply:
I'd been thinking the same thing! But not sure if it will be used other places, maybe through it in a resource related file? I'll go through our dirs and see if I can think of a good place for it.

Reply:
I’d also look and see if the mocks reuse this component at all. If they do, maybe `common/resources.tsx`?

Reply:
Put it in components/resources.tsx? Let me know if you want me to move it common/

Reply:
I think components/resources.tsx is a totally reasonable place for it!